### PR TITLE
upgrade notes: mention trailing slash in permalink; related to #4440

### PR DESCRIPTION
**Pull request by daattali**
https://talk.jekyllrb.com/t/github-pages-with-jekyll3-blog-posts-with-trailing-slash-no-longer-work/1877/4

### CODE REVIEW

File: site/_docs/upgrading/2-to-3.md [hunk 1]
```diff
@@ -99,4 +99,8 @@ This can be fixed by removing the following line from your `_config.yml` file:
 relative_permalinks: true
 {% endhighlight %}
 
+### Permalinks no longer automatically add a trailing slash
+
+In Jekyll 2, any URL constructed from the `permalink:` field had a trailing slash (`/`) added to it automatically.  Jekyll 3 no longer adds a trailing slash automatically to `permalink:` URLs. This can potentially result in old links to pages returning a 404 error. For example, suppose a page previously contained the YAML `permalink: /:year-:month-:day-:title` that resulted in the URL `example.com/2016-02-01-test/` (notice the trailing slash). In Jekyll 3, the URL for the same page is `example.com/2016-02-01-test`, and consequently any links to the old URL will result in a 404 error. In order to maintain the same URLs and avoid this problem, a trailing slash should be added to the `permanlink:` field, for example `permalink: /:year-:month-:day-:title/`.
```
Comment (parkr) [inline site/_docs/upgrading/2-to-3.md]: There's a `permanlink` misspelling in here. Would you mind fixing that up really quick? Thank you! Then I'll merge :)

**Comment by daattali** [comment]
@parkr typo fixed

**Comment by envygeeks** [comment]
@daattali please rebase to a single commit, thanks!

**Comment by envygeeks** [comment]
You don't need to close it :P, just `git rebase -i` and then `git push -f` and it'll update this pull.

**Comment by daattali** [comment]
Yeah but I was too lazy to clone it locally, I just did it all within GitHub (browser), and I don't think I could rebase in the browser

**Comment by parkr** [comment]
Superseded by #4455.

**Pull request closed without merging**